hw_3/script.js

```diff
@@ -0,0 +1,111 @@
+function checkArgLength(args) {
+    if (args.length !== 2) {
+        throw new Error("Not 2 arguments");
+    }
+}
+
+function checkMinMax(min, max) {
+    if (!Number.isInteger(min) || !Number.isInteger(max)) {
+        throw new Error("arguments are not Integer");
+    }
+    if (min > max) {
+        throw new Error("Min > Max");
+    }
+}
+
+function checkResult(result) {
+    if (result > Number.MAX_SAFE_INTEGER) {
+        return new Error("The result is greater than Number.MAX_SAFE_INTEGER")
```

Comment:
I think you've forgotten a template literal string here: 
```suggestion
        return new Error(`The value is greater than ${Number.MAX_SAFE_INTEGER}`)
```